RELEASE.md

```diff
@@ -39,6 +39,11 @@ Please add the following to your .travis.yml file:
   secure: "mQnECL+dXc5l9wCYl/wUz+AaYFGt/1G31NAZcTLf2RbhKo8mUenc4hZNjHCEv+4ZvfYLd/NoTNMhTCxmtBMz1q4CahPKLWCZLoRD1ExeXwRymJPIhxZUPzx9yHPHc5dmgrSYOCJLJKJmHiOl9/bJi123456="
 ```
 
+Make sure to encrypt against the public key of the main repository!  
```

Comment:
It would by good to add explicit `-r` on line 36 -> `travis encrypt -r org/repo BINTRAY_USER=adrianmole`